app/app.go

```diff
@@ -94,6 +96,12 @@ func New(options ...Option) *App {
 		}
 	}
 
+	if htmlTemplateWatcher, err := utils.NewHTMLTemplateWatcher("templates"); err != nil {
+		l4g.Error(utils.T("api.api.init.parsing_templates.error"), err)
```

Comment:
This is the same "error handling" as the template code had before (log, but otherwise ignore). I want to revisit this later though. This error is unrecoverable and should definitely cause the server to fail fast.